### PR TITLE
Added apple loop

### PR DESCRIPTION
**Pull request by bartex-bartex**
Eating apple feature in loop is now working.

### CODE REVIEW

**Comment by bartex-bartex** [comment]
Nice idea

File: loop.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+for i in range(10):
```
Comment (bartex-bartex) [inline loop.py:1]: Change variable 'i' to something else for clarity

**Comment by KasiaWozniak** [comment]
> Nice idea

+1

File: loop.py [hunk 2]
```diff
@@ -1,2 +1,2 @@
-for i in range(10):
-    print("Apple was eaten " + str(i))
+for x in range(10):
+    print("Apple was eaten " + str(x))
```
Comment (KasiaWozniak) [inline loop.py:2]: print??? PRINTF

**Review by KasiaWozniak — APPROVED** [review]
jest super

**Pull request merged**